packages/bsky/src/hydration/hydrator.ts

```diff
@@ -93,6 +94,7 @@ export type HydrationState = {
   labelers?: Labelers
   labelerViewers?: LabelerViewerStates
   labelerAggs?: LabelerAggs
+  knownFollowers?: KnownFollowers
```

Comment:
There should be a line down in `mergeStates()` to merge this new entry of hydration state.

Reply:
Ah geez, thanks 🤦